docs/knative-eventing-mesh/02-01-knative-eventing-mesh.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Architecture
+---
+
+The architecture of Knative Eventing Mesh relies heavily on the functionality provided by [Knative Eventing](https://knative.dev/docs/eventing/). To ensure a stable event flow between the Sender and the Subscriber, the Knative Eventing Mesh wires the Knative elements closely with the existing Kyma components.
+
+## Knative Eventing Mesh implementation
+
+This diagram shows how the Knative elements are implemented and wired with the already exisitng Kyma components, whereas the second diagram shows you how the events are processed by in Kyma.
+
+![Eventing implementation](./assets/eventing-mesh-implementation.svg)
+
+1. The user creates the [Application CR](https://kyma-project.io/docs/components/application-connector/#custom-resource-application) and binds it to the Namespace. 
```

Comment:
Please add blank line after every point and every subpoint  :) This is a good practice.

Reply:
👍